src/js/inputs/text.js

```diff
@@ -15,6 +15,7 @@
           '<div id="error-text" class="error-text"></div>' +
         ''),
         events:{
+          'change input': 'keyPress',
```

Comment:
they handle state of the component... for example.. if a form is automatically filled in by the browser.. .we want the input to recognize the change and update accordingly